src/components/KMultiselect/KMultiselect.vue

```diff
@@ -348,6 +349,10 @@ const props = defineProps({
     type: String,
     default: '',
   },
+  searchPlaceholder: {
+    type: String,
+    default: '',
+  },
```

Comment:
Shall we warn users when `collapsedContext` is `true` but `placeholder` is not empty?

Reply:
I don't think that's necessary: the default displayed text for both modes is _0 items selected_ - I don't think we need to specifically point that out.